stdlib/public/Differentiation/DifferentiationSupport.swift

```diff
@@ -1236,6 +1236,25 @@ extension Array where Element : Differentiable {
   }
 }
 
+extension Array where Element: Differentiable {
+  @derivative(of: append)
+  public mutating func _vjpAppend(_ element: Element) -> (
```

Comment:
Please make derivatives of public APIs be `@inlinable internal` since their symbols are not to be leaked.

Reply:
Wow, thanks for the catch. Done.